tests/cupy_tests/polynomial_tests/test_polyutils.py

```diff
@@ -0,0 +1,96 @@
+import unittest
+
+import numpy
+import pytest
+
+import cupy
+from cupy import testing
+
+
+@testing.parameterize(*testing.product({
+    'trim': [True, False]
+}))
+@testing.gpu
+class TestAs_series(unittest.TestCase):
```

Comment:
Please use camel case.
```suggestion
class TestAsSeries(unittest.TestCase):
```

Reply:
Could you add 2-dim ndarry input case?